x-pack/plugins/apm/public/components/shared/charts/latency_chart/index.tsx

```diff
@@ -99,7 +108,11 @@ export function LatencyChart({ height }: Props) {
           height={height}
           fetchStatus={latencyChartsStatus}
           id="latencyChart"
-          timeseries={latencyTimeseries}
+          customTheme={comparisonChartTheme}
+          timeseries={[
+            ...currentPeriod,
+            ...(comparisonEnabled && previousPeriod ? [previousPeriod] : []),
```

Comment:
are we using this flag in other places when deciding to pass the data to the component? not saying we shouldn't, but it should be consistent.

Reply:
@dgieselaar yes, the `comparisonEnabled` will be used on all charts to pass or not the previousPeriod data to the chart component.